near-sdk/src/collections/vector.rs

```diff
@@ -73,18 +73,7 @@ impl<T> Vector<T> {
             }
         } else {
             let lookup_key = self.index_to_lookup_key(index);
-            let last_lookup_key = self.index_to_lookup_key(self.len - 1);
-            self.len -= 1;
```

Comment:
This breaks the `Vector::len()` because we need to store the real decreased length in `self.len`

Reply:
`pop_raw` decreases length